MaxText/pyconfig.py

```diff
@@ -54,8 +54,16 @@ def validate_attention_type(s: str) -> None:
       "Invalid attention type was passed. Valid options ", valid_attention_types
     )
 
+def validate_opt_type(s: str) -> None:
```

Comment:
I'm actually fine to discover this later when we can't create the optimizer, not sure we need to duplicate the checking. (Since the dupe will fall behind, etc.)

Reply:
Good suggestion.

Reply:
Done.